public/assets/styles/main.css

```diff
@@ -21,8 +21,8 @@ a {
 
 #player-main-artist span, #player-main-title span, #player-main-album span {
   background: #292827;
-  color: #cccccc;
-  padding: 2px 5px;
+  color: #ccc;
+  padding: 2px 7px;
```

Comment:
perhaps consider not doing hard coded px as it may be difficult to refactor when turning this to mobile

Reply:
I'll put this on the refactor list as something to think about... May not be the best practice, but definitely something to look into!